db/models.py

```diff
@@ -1 +1,29 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(
+        "Describes what kind of bonus players can get from it",
+        max_length=255
+    )
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True, null=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255)
+    bio = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.PROTECT)
+    guild = models.ForeignKey(Guild, on_delete=models.PROTECT, null=True)
```

Comment:
Look at it this way: a guild is an optional feature. A player can create a guild, delete it, enter, or leave. If a guild owner decides to delete his guild, should they:
- a) be able to delete the guild, in which case all the players in it will have no guild (`SET_NULL`),
- b) be able to delete the guild, in which case all the players in it will be deleted (`CASCADE`), or
- c) not be able to delete the guild as long as there's at least one player in it (`PROTECT`)?

Reply:
ok, I understand